Makefile

```diff
@@ -11,4 +11,7 @@ tests:
 	PYTHONWARNINGS=ignore:ResourceWarning coverage run --source=dcplib \
 		-m unittest discover --start-directory tests --top-level-directory . --verbose
 
+additional_tests:
+	pytest -v tests/component_agents/test_analysis_agent.py
```

Comment:
this is hideous, probably standardize it to use `pytest -v` for all of them?

Reply:
or change these tests so that they can run via unittest?

Reply:
Just to echo what others have said: this project is already set up to use unittest, and I don't think pytest adds enough value to justify a switch. We should not have two different test frameworks running the tests alongside each other.

Reply:
All make sense, for now, I will switch back to unittest and update the test cases.